cmd/fleetctl/scripts_test.go

```diff
@@ -70,13 +70,20 @@ hello world
 -------------------------------------------------------------------------------------
 `
 
+	expectedQuietOutputSuccess := `hello world
+`
+	//expectedAsyncOutputSuccess := `423e5330-3591-440a-8908-fceb336a3643
+	//`
```

Comment:
Was this meant to be removed (or implemented)?

Reply:
I'll remove it for now